source/puppet/5.3/hiera_config_yaml_5.md

```diff
@@ -64,7 +64,9 @@ hierarchy:
       - 'common.yaml'
 ```
 
-Options for tweaking the hiera.yaml file:
+Note that `data_hash: yaml_data` and `datadir: data` are the default settings.
+
+A more advanced example of a hiera.yaml file:
```

Comment:
In the example below - it make sense to (again) comment out the defaults, so that it begins with this:
```
version: 5
# defaults:  # Used for any hierarchy level that omits these keys.
#   datadir: data         # This path is relative to hiera.yaml's directory.
#   data_hash: yaml_data  # Use the built-in YAML backend.
```